addons/account/models/account_account.py

```diff
@@ -414,12 +414,16 @@ def load(self, fields, data):
         with opening debit/credit. In that case, the auto-balance is postpone
         until the whole file has been imported.
         """
-        rslt = super(AccountAccount, self).load(fields, data)
-
         if 'import_file' in self.env.context:
+            # Never automatically delete imported accounts when upgrading modules.
+            rslt = super(AccountAccount, self.with_context(noupdate=True)).load(fields, data)
```

Comment:
Isn't this only the case with `base_import`?
I don't think it gets here with the upgrade?

Reply:
This PR is only intended to affect new imports, it doesn't do anything for records that were previously imported in the wrong way. This code indeed doesn't get called during an upgrade.

Reply:
But `base_import` doesn't change the data that comes from the l10n modules, I think we should still let users export/import if they want to